bin/webpack.js

```diff
@@ -183,6 +183,12 @@
 			type: "boolean",
 			group: DISPLAY_GROUP,
 			describe: "Show more details"
+		},
+		"info-verbosity": {
+			type: "string",
+			default: "info",
+			group: DISPLAY_GROUP,
+			describe: "Controls the output of lifecycle messaging e.g. Started watching files... (verbose, info, none)"
```

Comment:
What's the difference between `info` and `verbose` ? Seems to do the same thing.

Reply:
Well within this PR yes, but the next one should add two more messages they will be verbose only (see linked issues)

Reply:
Could you make it in this PR as well? Ask any of the maintainers if you got any questions!

Reply:
Probably it will stretch out for another month in that case, any good reason for that?

Reply:
Leaving code unfinished is a bad habit 💃

Reply:
But if you promise to respond quickly im on it! :)

Reply:
Pinky Promise 😆